self-core-impl/src/main/java/com/selfxdsd/core/managers/StoredProjectManager.java

```diff
@@ -304,10 +341,23 @@ public void assignedTasks(final Event event) {
                         task.deadline().toLocalDate()
                     ).getDays();
                     final int left = Period.between(
-                        LocalDateTime.now().toLocalDate(),
+                        this.dateTimeSupplier.get().toLocalDate(),
                         task.deadline().toLocalDate()
                     ).getDays();
-                    if(left <= time/2) {
+                    if (left < 0) {
```

Comment:
@amihaiemil 
is this check in days too "relaxed"? or should we use a stricter check like `left` in seconds/minutes <=0?

Reply:
@criske yeah, maybe we could take into account the hours/minutes as well. Can you leave a puzzle?